src/modules/transaction/components/SelectedUser.js

```diff
@@ -1,8 +1,12 @@
 import { h } from "preact";
 import styles from "./SelectedObject.scss";
 import clearIcon from "src/assets/icons/clear.svg";
+import Avatar from "src/assets/avatars/blank_avatar.jpg";
 
 const SelectedUser = ({ user, clearSelection }) => {
+  if(!user["avatar-url"] || user["avatar-url"] == "") {
```

Comment:
What exactly do you expect in this check? Can `user["avatar-url"]` produce an empty string or is it null?